lib/drizzlepac/createMedian.py

```diff
@@ -168,16 +168,22 @@ def _median(imageObjectList, paramDict):
         native_units = image.native_units
         if lthresh is not None:
             if proc_units.lower() == 'native':
-                if native_units.lower() == "counts":
+                if native_units.lower() == 'counts':
                     lthresh = lthresh * det_gain
-                    if native_units.lower() == "counts/s":
-                        lthresh = lthresh * img_exptime
+                elif native_units.lower() in ['counts/s', 'electrons/s']:
```

Comment:
I would suggest changing this to

    if native_units.lower() in ['counts/s', 'electrons/s']